stable/artifactory-ha/templates/artifactory-nfs-pvc.yaml

```diff
@@ -0,0 +1,89 @@
+{{- if eq .Values.artifactory.persistence.type "nfs" }}
+### Artifactory HA data
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ template "artifactory-ha.fullname" . }}-data-pv
+  labels:
+    app: {{ template "artifactory-ha.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    id: {{ template "artifactory-ha.name" . }}-data-pv
+    type: nfs-volume
+spec:
+  capacity:
+    storage: {{ .Values.artifactory.persistence.nfs.capacity }}
+  accessModes:
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  nfs:
+    server: {{ .Values.artifactory.persistence.nfs.ip }}
+    path: "{{ .Values.artifactory.persistence.nfs.haDataMount }}"
+    readOnly: false
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "artifactory-ha.fullname" . }}-data-pvc
+  labels:
+    app: {{ template "artifactory-ha.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    type: nfs-volume
+spec:
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: ""
+  resources:
+    requests:
+      storage: {{ .Values.artifactory.persistence.nfs.capacity }}
+  selector:
+    matchLabels:
+      id: {{ template "artifactory-ha.name" . }}-data-pv
+---
+### Artifactory HA backup
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ template "artifactory-ha.fullname" . }}-backup-pv
+  labels:
+    app: {{ template "artifactory-ha.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    id: {{ template "artifactory-ha.name" . }}-backup-pv
+    type: nfs-volume
+spec:
+  capacity:
+    storage: {{ .Values.artifactory.persistence.nfs.capacity }}
+  accessModes:
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  nfs:
+    server: {{ .Values.artifactory.persistence.nfs.ip }}
+    path: "{{ .Values.artifactory.persistence.nfs.haBackupMount }}"
+    readOnly: false
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "artifactory-ha.fullname" . }}-backup-pvc
+  labels:
+    app: {{ template "artifactory-ha.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    type: nfs-volume
+spec:
+  accessModes:
+  - ReadWriteOnce
+  storageClassName: ""
+  resources:
+    requests:
+      storage: {{ .Values.artifactory.persistence.nfs.capacity }}
+  selector:
+    matchLabels:
```

Comment:
Add `app` and `release` labels to selector for uniqueness in case the chart is installed multiple times on the same cluster.